server/settings.py

```diff
@@ -78,6 +81,14 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+if DOCKERFLOW_ENABLED:
+    INSTALLED_APPS.extend([
```

Comment:
I would just use `append` rather than `extend` here, though it's not that big a deal.

Reply:
Oof, yeah, I'll make that change.